homeassistant/generated/integrations.json

```diff
@@ -5167,6 +5167,10 @@
       "config_flow": false,
       "iot_class": null
     },
+    "sum": {
+      "config_flow": true,
+      "iot_class": "local_push"
```

Comment:
Should the `iot_class` not be `calculated`?
_`calculated`: The integration does not handle communication on its own, but provides a calculated result._